deltadb/src/catalog_export.c

```diff
@@ -160,3 +159,39 @@ void catalog_export_html_footer( struct link *l, struct jx_table *h, time_t stop
 {
 	link_printf(l,stoptime,"</table>\n");
 }
+
+void catalog_export_html_datetime_picker( struct link *l, time_t stoptime, time_t current) {
+	struct tm *t1 = gmtime(&current);
+	struct tm *t2 = gmtime(&current);
+	struct tm *t3 = gmtime(&current);
+
+	int year = t1->tm_year + 1900;
+	int month = t1->tm_mon + 1;
+	int day = t1->tm_mday;
+
+	t2->tm_mday--;
+	t2->tm_hour = t2->tm_min = t2->tm_sec = 0;
+	time_t yesterday = mktime(t2);
+
+	t3->tm_mday += 2;
```

Comment:
Not sure if midnight belongs to the prior day or the current day.
Perhaps using 12:01 will result in more clarity?

Reply:
Ah just realized it was because I had decremented the value a few lines earlier to get yesterdays value, so I needed to increment twice to reach tomorrows value.